modules/xml-sitemap.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Module Name: XML Sitemap
```

Comment:
Google Webmaster Tools/Search Console refers to them simply as "Sitemaps" too. +1 to dropping `XML` as jargon for the average user.